src/components/DetailsMetrics/detailsMetrics.utils.js

```diff
@@ -18,11 +18,44 @@ under the Apache 2.0 license is conditioned upon your compliance with
 such restriction.
 */
 import { capitalize, chain } from 'lodash'
+import {
+  CUSTOM_RANGE_DATE_OPTION,
+  PAST_24_HOUR_DATE_OPTION,
+  PAST_HOUR_DATE_OPTION,
+  PAST_MONTH_DATE_OPTION,
+  PAST_WEEK_DATE_OPTION
+} from '../../utils/datePicker.util'
+import { AVG, AVG_RAW, RAW_DATA_TOTAL, TOTAL } from '../../constants'
 
-const mlrunInfra = 'mlrun-infra'
 const metricsColorsByFullName = {}
 const usedColors = new Set()
 
+export const timeRangeMapping = {
+  [PAST_24_HOUR_DATE_OPTION]: 'last 24 day',
+  [PAST_WEEK_DATE_OPTION]: 'last week',
+  [PAST_MONTH_DATE_OPTION]: 'last month',
+  [PAST_HOUR_DATE_OPTION]: 'last hour',
+  [CUSTOM_RANGE_DATE_OPTION]: 'custom range'
+}
+
+export const calculatePercentageDrift = (previousTotalInvocation, currentTotalInvocation) => {
+  if (!previousTotalInvocation)
+    return {
+      className: 'drift_up',
+      percentageChange: 'N/A',
+      positive: true
+    }
+
+  const result =
```

Comment:
Use different variable name. result is very generic variable name

Reply:
done